### ISSUE TITLE
When deleting a version, the following error is displayed Error: ENOENT: no such file or directory, open

### ISSUE THREAD

**Issue by elmarcz** [sammwyy/minecraft-launcher-js#1]
![Screenshot_1](https://user-images.githubusercontent.com/83883889/216435126-e63eead9-cdc2-4246-8d0f-31cd737ee5a4.png)

**Comment by sammwyy** [comment]
Yes, that's because the library doesn't download the manifest (aka version.json) from the servers. Although it is not a bug, I will try to implement that function as a new feature.
